cassiopeia/data/utilities.py

```diff
@@ -412,3 +413,97 @@ def compute_phylogenetic_weight_matrix(
     np.fill_diagonal(W.values, 0)
 
     return W
+    
+@numba.jit(nopython=True)
+def net_relatedness_index(
+    dissimilarity_map: np.array, indices_1: np.array, indices_2: np.array
+) -> float: # pragma: no cover
```

Comment:
What does this comment do?

Reply:
I was thinking that it would ignore this function from the coverage report but I don't think it's the case. I'll get rid of it.

(The reason why I wanted to ignore it was apparently our coverage reports actually can't test functions wrapped in `numba`, so we have an artificially deflated coverage. Not a huge deal but something I was trying to counteract.)